restorers/dataloader/base/commons.py

```diff
@@ -12,6 +12,14 @@ def random_horiontal_flip(low_image, enhanced_image):
     )
 
 
+def random_unpaired_horiontal_flip(low_image):
```

Comment:
Just wanted to add that with the random flips ([left-right](https://www.tensorflow.org/api_docs/python/tf/image/random_flip_left_right)/[up-down](https://www.tensorflow.org/api_docs/python/tf/image/random_flip_up_down)) in the `tf.image` library, we can specify the same `seed` argument and both the images (low and enhanced) will be treated with the same augmentation.

We might want to benchmark the two APIs (the present pipeline with `tf.cond`, and the one with the `seed`).

Reply:
@ariG23498 Let's convert this comment into an issue.

Reply:
https://github.com/TrystAI/restorers/issues/59